### PR TITLE
Update env

### PR DESCRIPTION
**Pull request by nguyenhuudungz**

### CODE REVIEW

File: src/locales/zh-CN.po [hunk 1]
```diff
@@ -1677,14 +1677,14 @@ msgstr ""
 
 #: src/components/swapv2/TradeTypeSelection.tsx
 msgid "Maximum Return"
-msgstr "最大的收益"
+msgstr ""
```
Comment (viet-nv) [inline src/locales/zh-CN.po:1680]: Why do you remove this translate

**Review by viet-nv — CHANGES_REQUESTED** [review]

File: src/locales/zh-CN.po (hunk 1 above)
Comment (nguyenhuudungz) [reply src/locales/zh-CN.po:1680]: It is generated when I install packages.

Comment (viet-nv) [reply src/locales/zh-CN.po:1680]: Revert these translation files. Then run yarn i18n again.

Comment (nguyenhuudungz) [reply src/locales/zh-CN.po:1674]: Done.

**Comment by viet-nv** [comment]
updated on improve/elastic

**Pull request closed without merging**